CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unpublished
+* larva-patterns - Suffix relevant `c-text-input` component variables with `_attr`
+* larva-patterns - Deconflict `c-text-input` component variable names so that it's properly parsed by twig-to-php parser
```

Comment:
So...the issue here is that we have some logic in data controllers that looks for the prefix of the pattern in the names of properties that have data. When this happened before, my approach was to rename the component itself, instead of updating the prefix.

Technically, things _will_ work without the prefix, but it breaks a pattern of consistency which is a concern b/c we have things that rely on the conistency (also noting - this consistency is _not_ enforced in an automated way, so we are relying on referring to existing patterns).